developers/requesting-data-commitment-ranges.md

```diff
@@ -0,0 +1,26 @@
+# Requesting data commitment ranges
+
+To request proofs to be submitted to the Blobstream X contract at a
+different cadence, you can do one of the following:
+
+1. [RECOMMENDED] Run the Blobstream X operator with hosted proving, by
```

Comment:
maybe put **Recommended** in bold and then remove the the 1., 2., 3. and replace with section heading? like ### Recommended, ### Local Proving, ### Request proof onchain

Reply:
based on this and next few comments, I will rework this into headings. the lists are janky IMO too

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/7c5e7ca2432441486c7b8538a0226ebd527daf5f